spec/features/proposals/my_requests_needing_action_spec.rb

```diff
@@ -0,0 +1,39 @@
+describe "My requests needing action" do
```

Comment:
in feature specs, I  usually see `feature` / `scenario` instead of `describe` and `it` (although both work!)